build.ps1

```diff
@@ -61,9 +57,13 @@ else {
     # Install by channel or version
     $DotNetDirectory = "$TempDirectory\dotnet-win"
     if (!(Test-Path variable:DotNetVersion)) {
-        ExecSafe { & powershell $DotNetInstallFile -InstallDir $DotNetDirectory -Channel $DotNetChannel -NoPath }
+        # Octopus Modification to ensure TLS1.2 is enabled within the script

```

Comment:
Neat so we just set the protocol then call the script. That does seem like a better solution.